deployment/v2/4_createRollup.ts

```diff
@@ -300,6 +270,96 @@ async function main() {
+        // programVKey = bytes32(0)
+        programVKey = ethers.ZeroHash;
+        // programVKey = address(0)
+        verifierAddress = ethers.ZeroAddress;
+        if(consensusContract == "AggchainECDSA") {
+            initializeBytesCustomChain = utilsECDSA.encodeInitializeBytesAggchainECDSAv0(
+                createRollupParameters.aggchainParams.useDefaultGateway,
+                createRollupParameters.aggchainParams.ownedAggchainVKeys,
+                createRollupParameters.aggchainParams.aggchainVKeySelectors,
+                createRollupParameters.aggchainParams.vKeyManager,
+                adminZkEVM,
+                trustedSequencer,
+                gasTokenAddress,
+                trustedSequencerURL,
+                networkName
+            )
+        } else if(consensusContract.includes("FEP")) {
+            initializeBytesCustomChain = "0x";
+        } else {
+            throw new Error(`Aggchain ${consensusContract} not supported`);
+        }
+    } else {
+        // Verifier is only necessary if consensusContract !== Aggchain
+        let verifierContract;
+        if (realVerifier === true) {
+            if (consensusContract != "PolygonPessimisticConsensus") {
+                verifierName = `FflonkVerifier_${forkID}`;
+
+                const VerifierRollup = await ethers.getContractFactory(verifierName, deployer);
+                verifierContract = await VerifierRollup.deploy();
+                await verifierContract.waitForDeployment();
+            } else {
+                verifierName = "SP1VerifierPlonk";
+                const VerifierRollup = await ethers.getContractFactory(verifierName, deployer);
+                verifierContract = await VerifierRollup.deploy();
+                await verifierContract.waitForDeployment();
+            }
+        } else {
+            verifierName = "VerifierRollupHelperMock";
+            const VerifierRollupHelperFactory = await ethers.getContractFactory(verifierName, deployer);
+            verifierContract = await VerifierRollupHelperFactory.deploy();
+            await verifierContract.waitForDeployment();
+        }
+        verifierAddress = verifierContract.target;
+        initializeBytesCustomChain = "0x"
+        console.log("#######################\n");
+        console.log("Verifier name:", verifierName);
+        console.log("Verifier deployed to:", verifierAddress);
+
+        if (consensusContract == "PolygonPessimisticConsensus") {
+            rollupVerifierType = 1;
+            genesisFinal = ethers.ZeroHash;
+            programVKey = createRollupParameters.programVKey || ethers.ZeroHash;
+        } else {
+            rollupVerifierType = 0;
+            genesisFinal = genesis.root;
+            programVKey = ethers.ZeroHash;
+        }
+    }
+
+    await (
+        await rollupManagerContract.addNewRollupType(
+            PolygonconsensusContract.target,
+            verifierAddress,
+            forkID,
```

Comment:
Sanity check on all the params that check on the SC in this script: https://github.com/0xPolygonHermez/zkevm-contracts/blob/v10.0.0-rc.1/contracts/v2/PolygonRollupManager.sol#L464-L467

This is just to detect it before actually execute the call